src/commands/goal.ts

```diff
@@ -0,0 +1,38 @@
+import BaseCommand, { CommandArgs } from "./base_command";
+import { getDebugContext, sendOptionsMessage } from "../helpers/discord_utils";
+import { GameOption, getGuildPreference } from "../helpers/game_utils";
+import _logger from "../logger";
+const logger = _logger("limit");
+export const DEFAULT_GOAL = 0;
+
+export default class GoalCommand implements BaseCommand {
+    async call({ message, parsedMessage }: CommandArgs) {
+        const guildPreference = await getGuildPreference(message.guildID);
+        guildPreference.setGoal(parseInt(parsedMessage.components[0]));
+        await sendOptionsMessage(message, guildPreference, GameOption.GOAL);
+        logger.info(`${getDebugContext(message)} | Goal set to ${guildPreference.getGoal()}`);
+    }
+    validations = {
+        minArgCount: 1,
```

Comment:
To keep consistent with `,groups`, calling `,goal` by itself should reset to not having a goal